themes/classic/templates/customer/password-new.tpl

```diff
@@ -30,7 +30,18 @@
 
 {block name='page_content'}
     <form action="{$urls.pages.password}" method="post">
-
+    <ul class="ps-alert-error">
+      {foreach $errors as $error}
+        <li class="item">
+          <i>
+            <svg viewBox="0 0 24 24">
+              <path fill="#fff" d="M11,15H13V17H11V15M11,7H13V13H11V7M12,2C6.47,2 2,6.5 2,12A10,10 0 0,0 12,22A10,10 0 0,0 22,12A10,10 0 0,0 12,2M12,20A8,8 0 0,1 4,12A8,8 0 0,1 12,4A8,8 0 0,1 20,12A8,8 0 0,1 12,20Z"></path>
+            </svg>
+          </i>
+          <p>{$error}</p>
+        </li>
```

Comment:
Although we all like the design of the alert panel, it is not supposed to be implemented like this. This would be the only place we can see it.

I wonder if we just had to change the assigned variable with `$notifications.error`, to make it displayed on notifications.tpl
https://github.com/PrestaShop/PrestaShop/blob/44e907bb47f5a8a6f4e151ab058d5034a37610b2/themes/classic/templates/_partials/notifications.tpl#L29-L39

Reply:
Okay, I see that you reused the example in `password-email.tpl`, `password-infos.tpl` etc.

We made an error by not making this simple, and this needs to be simplified. Because this is outside the perimeter of this PR, I approve it.